tests/test_dump.py

```diff
@@ -18,6 +18,13 @@
 
 from . import get_betamax
 
+# A semi-random selection of "normal" headers not touched by sanitizing
+NORMAL_HEADERS = (
+    "Accept", "Accept-Encoding", "Host", "User-Agent", "Accept-Ranges",
+    "Cache-Control", "Content-Encoding", "Content-Length", "Content-Type",
+    "Date", "Etag", "Expires", "Last-Modified", "Server", "Vary", "X-Cache"
```

Comment:
What about custom headers? `OpenStack-Identity`, `X-My-Custom-Header`, etc.?

Reply:
This is used in a test to confirm that we are not redacting all headers. A test that covers all possible custom headers would be impossible, since that is a nearly infinite set. I decided to just use a semi-random selection of common headers.

If we are removing the `HeaderSanitizer`, then this point is moot anyway, since this test would go away too.